notebooks/profiling-scheduling.ipynb

```diff
@@ -0,0 +1,366 @@
+{
```

Comment:
If the notebook is added in that directory it might be worth to follow the right naming scheme.
Can you also provide binder to check how it renders.

Reply:
we have to decide if the notebook and the examples/scheduling/computation_time_experiments.py should be in the PR or not. @dbarbier could have views on this.

Reply:
My branch was not meant to be merged, individual commits should be cherry-picked and polished.
For instance we saw during our meeting that commit 3467728 was wrong, as mentioned in the log message, the test is buggy and must be fixed instead.   We were  also quite reluctant to merge 5239abf.

Reply:
Ok I did the cherry picking finally and will do another PR because I pushed in one another branch in my repo, so it is ok that I close this one and start a new ? thanks

Reply:
Sure, thanks.